app/views/content/non-uk-teachers/why-train-to-teach-transcript.md

```diff
@@ -0,0 +1,56 @@
+
+### Is it hard to get on to a teacher training course in England? 
+
+**Nellie D’Costa, Trainee science teacher:**
+I assumed that it would be hard because I come from India. But then it was quite straightforward and easy. 
+
+**Giorgio Tatananni, Trainee modern foreign languages teacher:**
+Getting a training place here was so much easier than I thought. 
+
+### How about sorting your visa and documents? 
+
+**Christobel Gondwe, Maths teacher:**
+Sorting out my visa was a lot easier than expected. The information on the UK Border Agency’s website was very comprehensive, and my school and training provider were very knowledgeable, so I had people holding my hand from start to finish. It was very easy to do. 
+
+### What support did you get? 
+
+**Nellie D’Costa, Trainee science teacher:**
+The support that I received when I applied for my training course was tremendous. I was supported at every step, right from the very beginning. Whenever I had a doubt, I would just call them up and they were more than happy to help me. 
+
+**Giorgio Tatananni, Trainee modern foreign languages teacher:**
+They are very thorough in your preparation. They never overload you with information. 
+
+**Christobel Gondwe, Maths teacher:**
+I felt that I had a lot of support. The application process was very straightforward. The process from start to finish was fantastic. 
+
+### Did the training meet your expectations? 
+
+**Giorgio Tatananni, Trainee modern foreign languages teacher:**
+The teacher training did not meet my expectation, it exceeded my expectation, simply because of all the support you get and all the help you have from your advisers and mentors. 
+
+**Nellie D’Costa, Trainee science teacher:**
+The training did meet my expectations and it was much more than I expected. It was really good because there are so many things that I didn’t know before, even though I was teaching, that I’ve learned here. So for me, it has been a fruitful experience. 
+
+### Anything else you’d like to tell us? 
+
+**Nellie D’Costa, Trainee science teacher:**
+The value of gaining a transferable qualification through training to teach in England is that you can go anywhere in the world and you can teach, and you can go in that part of the world and you can make a difference, not just in the lives of the children but also in the people around you. 
+
+### Help for non-UK trainees 
+
+If you are interested in training to teach languages or physics, you could be eligible for up to £39,000 in a bursary or scholarship and international relocation payment. 
+
+Teachers from 47 countries worldwide can now apply for English qualified teacher status – plus, languages and physics teachers could be eligible for a £10,000 international relocation payment. 
+
+To learn more, visit: https://getintoteaching.education.gov.uk/non-uk-teachers 
```

Comment:
@caitlin-reidy This line is causing a build failure

This is what it says in the video so I put it in the transcript - but of course that is creating an internal link which includes the domain name, which we can't do!

Could we say something like 'To learn more, **read our guidance for non-UK trainees and teachers**.

And then link to `/non-uk-teachers`? (NB reword as appropriate!)

I think that would be offering the same experience for sighted and non-sighted users then?